deepchecks/tabular/checks/model_evaluation/performance_bias.py

```diff
@@ -0,0 +1,549 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""The performance_disparity_report check module."""
+import itertools
+from typing import Callable, Tuple, Union
+
+import numpy as np
+import pandas as pd
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+from deepchecks.core import CheckResult, ConditionCategory, ConditionResult
+from deepchecks.core.checks import DatasetKind
+from deepchecks.core.errors import DeepchecksProcessError, DeepchecksValueError
+from deepchecks.tabular import Context, SingleDatasetCheck
+from deepchecks.utils.performance.partition import partition_column
+from deepchecks.utils.typing import Hashable
+
+
+class PerformanceBias(SingleDatasetCheck):
+    """
+    Check for performance disparities across subgroups, while optionally accounting for a control variable.
+
+    The check identifies "performance biases": large performance difference for a subgroup compared
+    to the baseline performance for the full population. It is intended to be used for fairness analyses.
+
+    Subgroups are defined by the categories of a "protected" feature. Numerical features are first binned
+    into quantiles, while categorical features are preserved as-is. The baseline score is the overall
+    score when all subgroups are combined. You can add conditions to flag performance differences outside
+    of given bounds. A visual display is also provided to identify subgroups with the largest performance
+    disparities.
+
+    Additionally, the analysis may be separated across the categories of a "control" feature. Numerical
+    features are binned and categorical features are re-binned into `max_number` categories. To account
+    for the control feature, baseline scores and subgroup scores are be computed within each of its
+    categories.
+
+    The check's display is a plot of the baseline score (black point) and subgroup scores (white point)
+    for each subgroup, faceted by control feature categories, and sorted by the size of the
+    performance disparity. The performance disparity is highlighted by a red line for negative differences
+    and a green line for positive differences. By default, only the top 3 categories of the control
+    feature with the largest performance disparities are displayed. Within each control category
+    subplot, the top 3 subgroups with the largest performance disparities are displayed.
```

Comment:
Some of this description doesn't belong in the docstring IMO. Should be as additional text to the graph itself like in correlation checks